website_sale_checkout_skip_payment/tests/test_checkout_skip_payment.py

```diff
@@ -15,6 +17,14 @@ def setUp(self):
         product = self.env.ref("product.product_product_4_product_template")
         product.optional_product_ids = [(6, 0, [])]
 
+    def test_checkout_skip_payment(self):
+        website = self.env.ref("website.website2")
+        with mock.patch(
```

Comment:
this is not needed anymore: use `website.tools.MockRequest` ctx manager :wink:

Reply:
Awesome! Thanks I didn't know that.